src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/processors/append/view_model.js

```diff
@@ -2,7 +2,15 @@ import Processor from '../base/view_model';
 
 export class Append extends Processor {
   constructor(processorId) {
-    super(processorId, 'append', 'Append');
+    super(
+      processorId,
+      'append',
+      'Append',
+      `Appends one or more values to an existing array if the field already exists
+and it is an array. Converts a scalar to an array and appends one or more
```

Comment:
Because I am using the backticks, if I align the additional lines of text with the first, all of the whitespace gets inserted into the string. What is the preferred method/format of setting multi-line string values?